NEWS.md

```diff
@@ -15,6 +15,11 @@
 
 * Improve documentation and advice for `deployApp(envVars...)`.
 
+* Removed support for publishing to Posit Cloud. (#1215)
+  * Existing Posit Cloud account records may be removed by using `rsconnect::removeAccount("yourname", "posit.cloud")`.
```

Comment:
```suggestion

  Existing Posit Cloud account records may be removed by using `rsconnect::removeAccount("yourname", "posit.cloud")`.
```

Let's use a second paragraph for this item rather than a sublist. We have previous examples of this in the release notes for 0.8.28.